core/src/main/java/com/amazon/opendistroforelasticsearch/sql/data/model/ExprValueUtils.java

```diff
@@ -166,12 +173,33 @@ public static Boolean getBooleanValue(ExprValue exprValue) {
   }
 
   /**
-   * Get {@link ZonedDateTime} from ExprValue of Date type.
+   * Get {@link LocalDate} from ExprValue of Date type.
    */
-  public static ZonedDateTime getDateValue(ExprValue exprValue) {
+  public static LocalDate getDateValue(ExprValue exprValue) {
     return exprValue.dateValue();
   }
 
+  /**
+   * Get {@link LocalTime} from ExprValue of Time type.
+   */
+  public static LocalTime getTimeValue(ExprValue exprValue) {
```

Comment:
could we get rid of the static method incase we already have the method impl in ExprValue.

Reply:
yes, will remove all similar static methods and use the impl instead